lib/components/viewers/nearby/stop.tsx

```diff
@@ -159,18 +159,23 @@ const Stop = ({
     setHoveredStop(undefined)
   }, [setHoveredStop])
 
-  const operator =
-    agencies.size === 1
-      ? transitOperators?.find((o) => o.agencyId === Array.from(agencies)[0])
-      : undefined
-
   if (nearbyViewConfig?.hideEmptyStops && patternRows.length === 0) return <></>
 
   return (
     <Card onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
       <CardHeader>
         <CardTitle>
-          <IconWithText icon={<Operator operator={operator} />}>
+          <IconWithText
+            icon={
+              <>
+                {transitOperators
+                  ?.filter((to) => Array.from(agencies).includes(to.agencyId))
+                  .map((to) => (
+                    <Operator key={to.agencyId} operator={to} />
```

Comment:
Can we filter out duplicate operators here? 
<img width="322" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/115499534/9bde3ff9-b8c8-4b72-9308-fabbb8ea140c">

Reply:
How would you suggest doing that? They're different operators with different IDs, but the same image. We could filter on image url? But isn't that a bit hacky

Reply:
Hmm in this example, even though they're different feeds, it looks like both the operators have the same `name`?

Reply:
Good idea! Filtering on name works great